justfile

```diff
@@ -1,46 +1,67 @@
 default:
   @just --list
 
-create-cluster:
-  kind create cluster --config ./crates/astria-celestia-jsonrpc-client/k8s/cluster-config.yml
+default_docker_tag := 'local'
 
-deploy-ingress-controller:
-  kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
+# Builds docker image for the crate. Defaults to 'local' tag.
+docker-build crate tag=default_docker_tag:
+  docker buildx build --load --build-arg TARGETBINARY={{crate}} -f containerfiles/Dockerfile -t {{crate}}:{{tag}} .
 
-wait-for-ingress-controller:
-  kubectl wait --namespace ingress-nginx --for=condition=ready pod --selector=app.kubernetes.io/component=controller --timeout=600s
+install-cli:
+  cargo install --path ./crates/astria-cli --locked
 
-start-celestia-jsonrpc-test-deployment:
-  kubectl apply -k crates/astria-celestia-jsonrpc-client/k8s/
+# Compiles the generated rust code from protos which are used in crates.
+compile-protos:
+  cargo run --manifest-path tools/protobuf-compiler/Cargo.toml
 
-wait-for-celestia-jsonrpc-test-deployment:
-  kubectl wait --namespace astria-celestia-jsonrpc-client-test deployment --for=condition=Available --selector=app.kubernetes.io/name=astria-celestia-jsonrpc-client-test --timeout=600s
+## Scripts related to formatting code
+default_lang := 'all'
 
-delete-cluster:
-  kind delete cluster --name astria-celestia-jsonrpc-client-test
+# Can format 'rust', 'toml', 'proto', or 'all'. Defaults to all
+fmt lang=default_lang:
+  @just _fmt-{{lang}}
 
-default_docker_tag := 'local'
+# Can lint 'rust', 'toml', 'proto', 'md' or 'all'. Defaults to all.
+lint lang=default_lang:
+  @just _lint-{{lang}}
 
-docker-build crate tag=default_docker_tag:
-  docker buildx build --load --build-arg TARGETBINARY={{crate}} -f containerfiles/Dockerfile -t {{crate}}:{{tag}} .
+_fmt-all: 
+  @just _fmt-rust 
+  @just _fmt-toml
+  @just _fmt-proto
 
-fmt-rust:
+@_lint-all: 
+  -just _lint-rust 
+  -just _lint-toml 
+  -just _lint-proto
+  -just _lint-md
```

Comment:
Error codes are swallowed on lint so all outputs will be listed

Reply:
I am not sure what this means.